### ISSUE TITLE
Import lists empty after update to 3.7.0

### ISSUE THREAD

**Issue by clivebeckett** [xperseguers/t3ext-ig_ldap_sso_auth#147]
After updating the extension to 3.7.0, the lists for importing LDAP users and groups (in the LDAP/SSO backend module) are empty, in both T3v10 and T3v11. Switching back to 3.6.0 (even under T3v11) lets the lists return.

Do I have to change anything in the settings?

<img width="399" alt="Screenshot 2022-07-29 at 10 57 27" src="https://user-images.githubusercontent.com/36693867/181724097-017c32b6-77e3-4b3f-bd69-40ee044294b1.png">

EDIT: Apparently logging in with LDAP credentials isn’t working either, even for those users added to the local be_users database before the extension update.

**Comment by someplace53** [comment]
I can confirm this for typo3 v10 with php 7.2. The problem is in Classes/Utility/LdapUtility.php Line 343:
```php
 if (!($this->searchResult = @ldap_search($this->connection, $baseDn, $filter, $attributes, $attributesOnly, $sizeLimit, $timeLimit, $dereferenceAliases, $controls))) {
```
If I remove the last parameter `$controls` of `ldap_search` the list contains users again and login is possible, too. I tried to `null` the parameter, which should be the [default](https://www.php.net/manual/en/function.ldap-search.php), but this does not work either

**Comment by dev-rke** [comment]
I can confirm this issue. TYPO3 v10, PHP 7.2, Update from ig_ldap_sso_auth 3.6.99 to 3.7.0.
Made a downgrade to 3.6.99.

**Comment by medarob** [comment]
Does the problem only occur with PHP 7.2 or also with 7.4 on TYPO3 10?
Which PHP Version was used in TYPO3 11 (7.4 or 8)?

I'm hesitate to update the extension to 3.7.x with this error :/

**Comment by clivebeckett** [comment]
> Does the problem only occur with PHP 7.2 or also with 7.4 on TYPO3 10? Which PHP Version was used in TYPO3 11 (7.4 or 8)?

To be honest, I cannot remember which PHP version my T3 was running on when I ran into the issue. On the other hand T3v11 only works in PHP 7.4 or newer, so I guess I was on PHP 7.4.

However, I just tested ig_ldap_sso_auth 3.7.0 again on T3v11.5.16 and PHPv7.4.3 – and it now works. I saw the list of users from the LDAP repository, was able to import one record as a backend user, and logging in with the LDAP credentials also works now. I have no idea where that issue came from in the first place.
